### DISCUSSION TITLE
Why are my commits on my personal GitHub account being authored by my enterprise GitHub account?

### DISCUSSION THREAD

**Discussion by epeltz33** [community/community#39704] [Enterprise]
### Select Topic Area

Question

### Body

Hell all, so I have two GitHub accounts. I have an enterprise account for a bootcamp I am enrolled in and I have my personal account. I am working on a project on my personal account not my enterprise account but when I looked at my commit details it is saying that most of my commits are authored by my enterprise GitHub account not my personal GitHub account, even though I am logged in and the repo is created in my personal account not my enterprise account. How do I fix this to only  let my personal GitHub account commit to the repo ? Here is a link to my[ repo ](https://github.com/epeltz33/Budget_Buddy1). My username for my personal GH account is epeltz33, and my username for my enterprise GH account is Eric-Peltzman33.

**Comment by radar-blip** [comment]
This happens sometimes if you're using the same email address for both accounts.
Commits show up on both accounts because your emails are connected.
